src/PrestaShopBundle/Entity/Repository/TabRepository.php

```diff
@@ -137,7 +137,8 @@ public function getIdByClassName(string $className): int
         if (!isset($this->cachedTabIds[$className])) {
             $result = $this->createQueryBuilder('t')
                 ->select('t.id, t.className')
-                ->where('t.className = :className')
+                // Use binary to force a case-sensitive comparison (HOME and Home are different)
```

Comment:
It solves the problem, but why would anyone call their tab "Home" and not something more specific like ModuleNameHome? 🤔

Reply:
I think the lack of specificity in the naming may lead to more issues in other places of the core 🤔